packages/public/frontend/next.config.js

```diff
@@ -8,10 +8,8 @@ const withBundleAnalyzer = require('@next/bundle-analyzer')({
 
 module.exports = withCSS({
   target: 'serverless',
-  generateBuildId: async () => {
-    // For example get the latest git commit hash here
-    return require('./package.json').version
-  },
+  generateBuildId: async () =>
+    process.env.BUILD_ID ? process.env.BUILD_ID : null,
```

Comment:
Why do you prefer that over the version?

Reply:
The asset url already contains the version so it seems to be unnecesary to include it again

Reply:
True. But if I understood the documentation correctly, `null` means that the usual non-deterministic nextjs build id is used. So shouldn't we pass something constant instead?

Reply:
The build command in package.json sets it to the static string `current`, is this sufficient?

Reply:
Ah, didn't see that. That's fine. I'd even set the build id here directly since we don't have a reason to change that up anyways.